tests20/python_client/base/index_wrapper.py

```diff
@@ -10,41 +10,33 @@
 class ApiIndexWrapper:
     index = None
 
-    def index_init(self, collection, field_name, index_params, name="", check_res=None, check_params=None, **kwargs):
+    def init_index(self, collection, field_name, index_params, name="",check_task=None, check_params=None, **kwargs):
```

Comment:
check_params -> check_items, and we shall pass the check items and expectations into the check_items